README.md

```diff
@@ -59,6 +60,31 @@ Other supported formats are listed below.
 * `Workstation ID` - The workstation name (default is the host name)
 * `ApplicationIntent` - Can be given the value `ReadOnly` to initiate a read-only connection to an Availability Group listener. The `database` must be specified when connecting with `Application Intent` set to `ReadOnly`.
 
+### Kerberos Active Directory authentication outside Windows
+The package supports authentication via 3 methods.
```

Comment:
what's the default approach used if the connection string doesn't specify one explicitly? If a keytab file exists in some default location will it be used without having to specify it in the connection string?

Reply:
@shueybubbles as of now, no. The user needs to explicitly specify the locations for keytab/cache. 

A default location can be added. Should we?

Reply:
the MIT doc itself says to put it in /etc by default, or set environment variable KRB5_CONFIG. Couldn't we just rely on those too? I'm a bit surprised the jcmturner/gokrb5 module isn't already doing that

Reply:
I'd expect someone to be able to set the kerberos environment variables [https://web.mit.edu/kerberos/krb5-1.12/doc/admin/env_variables.html](https://web.mit.edu/kerberos/krb5-1.12/doc/admin/env_variables.html) and not have to put them all explicitly in the connection string here.

Reply:
I read [https://github.com/jcmturner/gokrb5/blob/master/config/krb5conf.go](https://github.com/jcmturner/gokrb5/blob/master/config/krb5conf.go) and have a recommendation for a fallback model for the kr5b.conf file location.
1. Specified in the connection string
2. Check KRB5_CONFIG environment variable
3. Check for /etc/krb5.conf
4. just use config.NewConfig()

If the user doesn't specify paths to keytab or cache, they probably can rely on the defaults defined in krb5.conf.
If there's no krb5.conf file in a default location or the environment variables aren't set, the authentication will fail but that's by design.

Does this seem reasonable?

Reply:
@shueybubbles yes, it does. will make the changes.